src/main/java/uk/gov/hmcts/reform/sendletter/health/RedisHealthIndicator.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.sendletter.health;
+
+import com.google.common.base.Strings;
+import org.redisson.client.RedisClient;
+import org.redisson.client.RedisClientConfig;
+import org.redisson.client.RedisConnectionException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class RedisHealthIndicator implements HealthIndicator {
+
+    private final RedisClientConfig config;
+
+    public RedisHealthIndicator(
+        @Value("${redis.host}") String host,
+        @Value("${redis.port}") int port,
+        @Value("${redis.password}") String password
+    ) {
```

Comment:
maybe possible to refactor `RedisConfiguration` to include client in there? Also _hide_ it under the feature flag so to prevent autoconstructing on runtime

Reply:
Health indicator uses `org.redisson.client.RedisClientConfig`, while cache uses `org.redisson.config.Config`

Reply:
hmm. what's the reason of the difference? is it possible to convert between one and the other?

Reply:
`RedisClient` is a low level client and it allows creating connections while the `RedissonClient` does not :/